src/clib/lib/enkf/ensemble_config.cpp

```diff
@@ -821,40 +821,47 @@ ensemble_config_forward_init(const ensemble_config_type *ens_config,
         for (auto &config_pair : ens_config->config_nodes) {
             enkf_config_node_type *config_node = config_pair.second;
             if (enkf_config_node_use_forward_init(config_node)) {
-                enkf_node_type *node = enkf_node_alloc(config_node);
-                enkf_fs_type *sim_fs = run_arg_get_sim_fs(run_arg);
-                node_id_type node_id = {.report_step = 0, .iens = iens};
-
-                /*
-           Will not reinitialize; i.e. it is essential that the
-           forward model uses the state given from the stored
-           instance, and not from the current run of e.g. RMS.
-        */
-
-                if (!enkf_node_has_data(node, sim_fs, node_id)) {
-                    if (enkf_node_forward_init(
-                            node, run_arg_get_runpath(run_arg), iens))
-                        enkf_node_store(node, sim_fs, node_id);
-                    else {
-                        char *init_file = enkf_config_node_alloc_initfile(
-                            enkf_node_get_config(node),
-                            run_arg_get_runpath(run_arg), iens);
-
-                        if (init_file && !fs::exists(init_file))
-                            error_msg = fmt::format(
-                                "File not found: {} - failed to initialize "
-                                "node: {}\n",
-                                init_file, enkf_node_get_key(node));
-                        else
-                            error_msg =
-                                fmt::format("Failed to initialize node: {}\n",
-                                            enkf_node_get_key(node));
-
-                        free(init_file);
-                        result = LOAD_FAILURE;
+
+                if (enkf_config_node_get_impl_type(config_node) == GEN_KW) {
+                    result = LOAD_FAILURE;
+                    error_msg = "Loading GEN_KW created in the forward model "
```

Comment:
Perhaps,

"Loading GEN_KW from files created by the forward model is not supported."